webpack.config.js

```diff
@@ -7,7 +7,7 @@ const HTMLWebpackPlugin = require('html-webpack-plugin');
 module.exports = {
   context: path.resolve(__dirname, 'src'),
   output: {
-    path: path.resolve(__dirname, './dist'),
+    path: path.resolve(__dirname, './public'),
```

Comment:
This needs to be changed because it's the hosting-sensitive setting that prevents develop and master branches to be built.